api/Element.json

```diff
@@ -6353,6 +6353,238 @@
           }
         }
       },
+      "transitioncancel_event": {
+        "__compat": {
+          "description": "<code>transitioncancel</code> event",
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Element/transitioncancel_event",
+          "spec_url": [
+            "https://drafts.csswg.org/css-transitions/#transitioncancel",
+            "https://drafts.csswg.org/css-transitions/#ref-for-dom-globaleventhandlers-ontransitioncancel"
+          ],
+          "support": {
+            "chrome": [
+              {
+                "version_added": "87"
+              },
+              {
+                "version_added": "74",
+                "partial_implementation": true,
+                "notes": "The <code>ontransitioncancel</code> event handler property is not supported. To listen to this event, use <code>element.addEventListener('transitioncancel', function() {});</code>."
+              }
+            ],
+            "chrome_android": "mirror",
+            "edge": "mirror",
+            "firefox": {
+              "version_added": "53"
+            },
+            "firefox_android": "mirror",
+            "ie": {
+              "version_added": false
+            },
+            "oculus": "mirror",
+            "opera": "mirror",
+            "opera_android": "mirror",
+            "safari": [
+              {
+                "version_added": "13.1"
+              },
+              {
+                "version_added": "12",
+                "partial_implementation": true,
+                "notes": "The event handler exists but will never be called."
```

Comment:
Can you match the wording we have elsewhere? "Although the <code>onmessageerror</code> property is supported, the <code>messageerror</code> event is never fired. See <a href='https://webkit.org/b/171216'>bug 171216</a>."